docs/v0.1.0/api-reference/importer-props.mdx

```diff
@@ -100,3 +106,14 @@ Sets the maximum allowed file size in bytes. Files exceeding this limit will be
 
 **Description:**  
 Provides a custom function to generate suggested mappings based on CSV headers.
+
+---
+
+## onSuccessRedirectUrl
```

Comment:
Shouldn't this be a callback? Wouldn't that be more customizable? Then integrator could just navigate to new page if they want to?